### PR TITLE
GlauberNBD fitter in O2Physics

### PR DESCRIPTION
**Pull request by ddobrigk**
This PR is part of an effort to make all Pb-Pb tools available inside O2Physics. With the class added here, all Glauber+NBD exercises will be doable inside O2Physics and the class will then supersede the development of the one in AliPhysics. Still missing: the correct usage of the anchor points determined with this class in the calibrator + the corresponding macros.

### CODE REVIEW

File: Common/Tools/Multiplicity/CMakeLists.txt [hunk 1]
```diff
@@ -16,3 +16,11 @@ o2physics_add_library(multCalibrator
...
                       HEADERS multCalibrator.h
                       LINKDEF multCalibratorLinkDef.h)
+
+o2physics_add_library(multGlauberNBDFitter
+                      SOURCES multGlauberNBDFitter.cxx
+                      PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore)
+
+o2physics_target_root_dictionary(multGlauberNBDFitter
+                      HEADERS multGlauberNBDFitter.h
+                      LINKDEF multGlauberNBDFitterLinkDef.h)
```
Comment (victor-gonzalez) [inline Common/Tools/Multiplicity/CMakeLists.txt:26]: It is better to have a single `LinkDef.h` for the whole directory and then a single dictionary which only built once

File: Common/Tools/Multiplicity/multGlauberNBDFitter.cxx [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+  fGlauberNBD->SetParameter(2, ff);
+  fGlauberNBD->SetParameter(3, fnorm);
+}
+//________________________________________________________________
+multGlauberNBDFitter::~multGlauberNBDFitter()
+{
+  // Destructor
+  if (fNBD) {
+    delete fNBD;
+    fNBD = 0x0;
```
Comment (victor-gonzalez) [inline Common/Tools/Multiplicity/multGlauberNBDFitter.cxx:124]: If you allocate with `[]` you have to use `delete [] fNBD`
The same for the other arrays

Comment (victor-gonzalez) [reply Common/Tools/Multiplicity/multGlauberNBDFitter.cxx:124]: OK!
My fault! You did it correctly for the array cases!
Sorry!

File: Common/Tools/Multiplicity/multGlauberNBDFitterLinkDef.h [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class multGlauberNBDFitter + ;
```
Comment (victor-gonzalez) [inline Common/Tools/Multiplicity/multGlauberNBDFitterLinkDef.h:16]: As said above, use a single `LinkDef.h` and incorporate all definitions there

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
Once the `LinkDef.h` file is incorporated we move forward

**Comment by victor-gonzalez** [comment]
Fantastic!!!
I believe is better to follow the usual structure of one single `LinkDef.h` file per directory

File: Common/Tools/Multiplicity/multGlauberNBDFitterLinkDef.h (hunk 3 above)
Comment (ddobrigk) [reply Common/Tools/Multiplicity/multGlauberNBDFitterLinkDef.h]: Done! I now created a multTools lib which incorporates all multiplicity related tools :-)

**Review by victor-gonzalez — APPROVED** [review]

**Pull request merged**